ComponentAssemblies/README.md

```diff
@@ -1,5 +1,8 @@
 ## Component Assemblies
 Directories that contain resources used to populate the public Catalog discovery UI introduced in 1.42.
+
+These Assemblies contain resources and are imported into the Vantiq public clouds as a Public Catalog. That allows these resources to be easily discovered in the UI builders.  The possible types of resources are Client, Theme, App, and GenAI Flow, and they show up using the Discover button in the Component palette in the Client Builder, Service Builder, and GenAI Builder, and in the Theme tab of the Client Builder.
```

Comment:
since it now says "resources", should add component to the next sentence
```suggestion
These Assemblies contain resources and are imported into the Vantiq public clouds as a Public Catalog. That allows these resources to be easily discovered in the UI builders.  The possible types of resources are Client Components, Themes, App Components, and GenAI Flow Components, and they show up using the Discover button in the Component palette in the Client Builder, Service Builder, and GenAI Builder, and in the Theme tab of the Client Builder.
```